### PR TITLE
Added opmon metric checking in tpg_state_collection_test.py and minimal_system_quick_test.py

### PR DESCRIPTION
**Pull request by bieryAtFnal**
# Description

These changes are for `fddaq-v5.6.0`... 

This PR is part of DUNE-DAQ/daq-deliverables#202 which describes the addition of opmon metric checking to integration tests.  The tests that were modified were `tpg_state_collection_test.py` and `minimal_system_quick_test.py`.

These changes build on ones in the `opmonlib` and `integrationtest` repos.

To test these changes one would include the modified repositories in a local software area and run the modified tests.

## Type of change

- [ ] Documentation (non-breaking change that adds or improves the documentation)
- [x] New feature or enhancement (non-breaking change which adds functionality)
- [ ] Optimization (non-breaking change that improves code/performance)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (whatever its nature)

## Testing checklist

- [ ] Unit tests pass (e.g. `dbt-build --unittest`)
- [x] Minimal system quicktest passes (`pytest -s minimal_system_quick_test.py`)
- [x] Full set of integration tests pass (`daqsystemtest_integtest_bundle.sh`)
- [ ] Python tests pass if applicable (e.g. `python -m pytest`)
- [ ] Pre-commit hooks run successfully if applicable (e.g. `pre-commit run --all-files`)

## Further checks

- [x] Code is commented where needed, particularly in hard-to-understand areas
- [ ] Code style is correct (`dbt-build --lint`, and/or see https://dune-daq-sw.readthedocs.io/en/latest/packages/styleguide/)
- [ ] If applicable, new tests have been added or an issue has been opened to tackle that in the future.

### CODE REVIEW

**Review by eflumerf — APPROVED** [review]
Additional tests pass

**Comment by eflumerf** [comment]
It might be worth-while to add the parameter names to the function calls (at least the min and max) to clarify what those hard-coded numbers are

**Comment by bieryAtFnal** [comment]
> It might be worth-while to add the parameter names to the function calls (at least the min and max) to clarify what those hard-coded numbers are

Great idea.  I've added that.

File: integtest/tpg_state_collection_test.py [hunk 1]
```diff
@@ -294,3 +295,37 @@ def test_tpstream_files(run_nanorc):
...
+    # *** Check that the pedestal subtraction processor metrics are being produced as expected.
+    # DLH-0, 'accum' metrics
+    metric_key_list = [session_name, "ru-det-conn-0", "DLH-0", "WIBEthFrameProcessor", "def-wib-processor", "datahandlinglibs.TPGProcessorInfo", "*", "accum"]
+    all_ok &= opmon_metric_checks.check_metric_sample_count(metric_data, metric_key_list, min_count=1)
+    all_ok &= opmon_metric_checks.check_metric_value_sum(metric_data, metric_key_list, min_value_sum=1)
+
+    # DLH-0, 'pedestal' metrics
+    metric_key_list = [session_name, "ru-det-conn-0", "DLH-0", "WIBEthFrameProcessor", "def-wib-processor", "datahandlinglibs.TPGProcessorInfo", "*", "pedestal"]
+    all_ok &= opmon_metric_checks.check_metric_sample_count(metric_data, metric_key_list, min_count=1)
+    all_ok &= opmon_metric_checks.check_metric_value_sum(metric_data, metric_key_list, min_value_sum=0, max_value_sum=0)
```
Comment (xinyue-uoft) [inline integtest/tpg_state_collection_test.py:328]: Would it be helpful to add a small note here like “In this integration test setup, all pedestal metric values are expected to fall to zero after a short initial settling period”? Since the pedestal values are initialized as non-zero constants but updates to zero before the first opmon snapshot. This could help clarify why the test checks for an exact sum of 0.

**Review by xinyue-uoft — APPROVED** [review]
Everything else looks good to me!

File: integtest/tpg_state_collection_test.py (hunk 1 above)
Comment (bieryAtFnal) [reply integtest/tpg_state_collection_test.py:328]: Great suggestion.  That has been done.
